examples/openstack/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```

Comment:
Instead of `'../../'` I used `File.expand_path("#{File.dirname(__FILE__)}/../..")` in order to have 1 less place which depends on current directory.